omaha/trivial_server_test.go

```diff
@@ -82,6 +83,37 @@ func TestTrivialServer(t *testing.T) {
 		t.Fatalf("failed to parse body: %v", err)
 	}
 
+	// Should get zero update because the version is already the latest.
+	if len(resp.Apps) != 1 ||
```

Comment:
Is this `if` missing the body? Or is is implicitly trying to panic on wrong responses?